projects/schematics/src/migrations/3_0/constructor-deprecations/data/user-order.service.migration.ts

```diff
@@ -21,8 +22,22 @@ export const USER_ORDER_SERVICE_MIGRATION: ConstructorDeprecation = {
       className: USER_ID_SERVICE,
```

Comment:
this came from another PR, but this is not supposed to be here, so remove it and fix please 
What you had before was good as in release/2.0.x, there was no UserIdService as dependency

Reply:
Done